common/web/keyboard-processor/src/text/codes.ts

```diff
@@ -98,13 +98,6 @@ const Codes = {
         if(Codes.keyCodes[keyID] >= 50000) { // A few are used by `sil_euro_latin`.
           return true; // is a 'K_' key defined for layer shifting or 'control' use.
         }
-        // Refer to text/codes.ts - these are Keyman-custom "keycodes" used for
-        // layer shifting keys.  To be safe, we currently let K_TABBACK and
-        // K_TABFWD through, though we might be able to drop them too.
-        const code = Codes[keyID];
-        if(code > 50000 && code < 50011) {
-          return true;
-        }
```

Comment:
Once I stopped and looked at this... "wait, why do we have two near-identical checks"?  And... "when did this happen?"

The answer to the latter question: it happened in #7242.  I don't know what I was thinking then.  It's clear we should only have one of the two.

Granted, settling on the one that I did kind of implies we should also ignore `K_TAB` for consistency.  (I haven't made this change yet, though.)